apps/web-portal/lib/WebPortalExpressApp.mjs

```diff
@@ -1,18 +1,18 @@
 import assert from 'assert'
 import express from 'express'
-import * as RestApiV1 from './rest-api-v1'
 import {makeRunner} from '@zorko-io/util-use-case'
+import logger from './logger'
+import * as RestApiV1 from './rest-api-v1'
 
-// TODO: Handle process start/stop
-// - add logging
-// - replace console.log
-// label: enhancement
 export class WebPortalExpressApp {
   /**
    * @type {Express}
    */
 
+  #app = null
+
   #http = null
+
   #config = null
 
   /**
```

Comment:
extend jsdocs and assert checks for `logger` and 'process'

also, I've found a good practice to pass defaults like

```
constuctor(context = { process, log: new MockLogger()}){

}
```
One of the reason is the building of integration tests over that application

Reply:
done